src/bench/rpc_blockchain.cpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2016-2019 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <bench/bench.h>
+
+#include <validation.h>
+#include <streams.h>
+#include <consensus/validation.h>
+#include <rpc/blockchain.h>
+
+#include <univalue.h>
+
+namespace block_bench {
+#include <bench/data/block413567.raw.h>
```

Comment:
This is a huge amount of data which is already part of the executable through `checkblock.cpp`. I think you need to make sure that this is only compiled and linked once.

Reply:
Can you suggest how I can resolve this? Define namespace block_bench in header file? (not sure that this will help) Move namespace to own cpp file?

Reply:
@laanwj nice catch, I wasn't aware of the generation rule in the makefile.

@fanatid I'm going to push a commit with a suggestion.

Reply:
@fanatid see #16299, you might want to rebase with it.

Reply:
Thanks! Rebased on top of #16299